project_W/app.py

```diff
@@ -63,32 +65,29 @@ def decode_key_loader(_jwt_header, jwt_data):
         # If the user doesn't exist then the token will fail anyways
         return JWT_SECRET_KEY
 
-    @app.post("/api/signup")
+    @app.post("/api/user/signup")
```

Comment:
Seeing as the prefixes for job/runner management are `/api/jobs` and `/api/runners`, should the user management prefix be `/api/users`? Or should we make the other prefixes singular as well?

Reply:
Yeah I know what you mean. What I had in my mind is that all these routes just apply to one user at a time (for example: /api/user/info returns the userinfo of this one specific user) while the /api/jobs api often describes multiple jobs at a time (like /api/jobs/list which returns the list of all jobs of a user). But you are right, it does seem inconsistent. Should I change it?

Reply:
I don't have a strong opinion on it, just do what seems most logical to you I guess.